### PR TITLE
ar - added jobMilkCows place holder for admin jobs

### PR DESCRIPTION
**Pull request by AlexanderRobles-CS**
In this PR, I created a milkTheCows job for the admin user. In order to see the job on local host, you must run the front and backend and create a common in swagger. Then you go to admin-> manage jobs -> milk the cows job. 

image

### CODE REVIEW

File: frontend/src/main/components/Jobs/jobMilkCows.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function jobMilkCows() {
```
Comment (briggsrr) [inline frontend/src/main/components/Jobs/jobMilkCows.js:1]: I think that the file name and function name should be JobMilkCows as referenced in the following form

Comment (briggsrr) [reply frontend/src/main/components/Jobs/jobMilkCows.js:1]: But looks good !

Comment (cyril-wang) [reply frontend/src/main/components/Jobs/jobMilkCows.js:1]: oh sorry that was my fault, i suggested that. We can change that then

Comment (AlexanderRobles-CS) [reply frontend/src/main/components/Jobs/jobMilkCows.js:1]: Resovled.

**Review by Geivet — DISMISSED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

File: frontend/src/main/pages/AdminJobsPage.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { useBackend } from "main/utils/useBackend";
 import Accordion from 'react-bootstrap/Accordion';
 import TestJobForm from "main/components/Jobs/TestJobForm";
 import JobComingSoon from "main/components/Jobs/JobComingSoon";
+import JobMilkCows from "main/components/Jobs/JobMilkCows";
```
Comment (pconrad) [inline frontend/src/main/pages/AdminJobsPage.js:8]: For consistency, we should probably have a consistent naming convention between "TestJobForm" and "MilkCowsJobForm".   Eventually, this component, will have to become a form in order to be functional.   Could you rename it?

**Review by pconrad — CHANGES_REQUESTED** [review]

**Review by cyril-wang — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**